src/common/function/src/scalars/geo/h3.rs

```diff
@@ -633,22 +633,24 @@ impl Function for H3CellParent {
     }
 }
 
-/// Function that checks if two cells are neighbour
+/// Function that returns children cell list
 #[derive(Clone, Debug, Default, Display)]
 #[display("{}", self.name())]
-pub struct H3IsNeighbour;
+pub struct H3CellToChildren;
```

Comment:
Is this a breaking change?

Reply:
We don't have a release and announcement about this feature so I assume it's ok